chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/CertificationBodyManagerImpl.java

```diff
@@ -97,7 +97,7 @@ public CertificationBodyDTO create(final CertificationBodyDTO acb)
     @PreAuthorize("@permissions.hasAccess(T(gov.healthit.chpl.permissions.Permissions).CERTIFICATION_BODY, "
             + "T(gov.healthit.chpl.permissions.domains.CertificationBodyDomainPermissions).UPDATE, #acb)")
     @CacheEvict(value = {
-            CacheNames.COLLECTIONS_DEVELOPERS, CacheNames.GET_DECERTIFIED_DEVELOPERS
+            CacheNames.COLLECTIONS_DEVELOPERS, CacheNames.GET_DECERTIFIED_DEVELOPERS, CacheNames.COLLECTIONS_LISTINGS
     }, allEntries = true)
     // listings collection is not evicted here because it's pre-fetched and handled in a listener
```

Comment:
remove this comment too about pre-fetching too please

Reply:
done